### PR TITLE
add example for atlas-aws vpc peering

### PR DESCRIPTION
**Pull request by nikhil-mongo**
## Description

Add example for atlas-aws vpc peering

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    regions_config {
+      region_name     = "US_EAST_1"
+      electable_nodes = 3
+      priority        = 7
+      read_only_nodes = 0
+    }
+  }
+  provider_backup_enabled      = true
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.2"
```
Comment (themantissa) [inline examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf]: Just a comment for later but we should try to make this always the latest version. Not a blocker though.

File: examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  accepter_region_name   = var.aws_region
+  project_id             = mongodbatlas_project.aws_atlas.id
+  container_id           = mongodbatlas_network_container.atlas_container.container_id
+  provider_name          = "AWS"
+  route_table_cidr_block = aws_vpc.primary.cidr_block
+  vpc_id                 = aws_vpc.primary.id
+  aws_account_id         = var.aws_account_id
+}
+
+resource "mongodbatlas_project_ip_whitelist" "test" {
```
Comment (themantissa) [inline examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf]: Whitelist is deprecated and being removed, this needs to be access list.

File: examples/MongoDB-Atlas-AWS-VPC-Peering/Readme.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+... or follow as in the `variables.tf` file and create **terraform.tfvars** file with all the variable values and make sure **not to commit it**.
+
+**2\. Review the Terraform plan.**
+
+Execute the below command and ensure you are happy with the plan.
+
+``` bash
+$ terraform plan
+```
+This project currently does the below deployments:
```
Comment (themantissa) [inline examples/MongoDB-Atlas-AWS-VPC-Peering/Readme.md]: very small nit, replace does with creates.

File: examples/README.md [hunk 4]
```diff
@@ -1,19 +0,0 @@
-# MongoDB Atlas Provider Examples
```
Comment (themantissa) [inline examples/README.md]: Don't think we want to remove this?

**Review by themantissa — CHANGES_REQUESTED** [review]
Looks good, just need a few changes!

File: examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf (hunk 1 above)
Comment (nikhil-mongo) [reply examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf]: fixed

File: examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf (hunk 2 above)
Comment (nikhil-mongo) [reply examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf]: fixed

File: examples/MongoDB-Atlas-AWS-VPC-Peering/Readme.md (hunk 3 above)
Comment (nikhil-mongo) [reply examples/MongoDB-Atlas-AWS-VPC-Peering/Readme.md]: fixed

File: examples/README.md (hunk 4 above)
Comment (nikhil-mongo) [reply examples/README.md]: fixed

**Review by themantissa — APPROVED** [review]
LGTM!

**Pull request merged**